manager/apis/app/v1beta1/fybrikapplication_types.go

```diff
@@ -116,10 +116,6 @@ type SecretRef struct {
 
 // DatasetDetails holds details of the provisioned storage
 type DatasetDetails struct {
-	// Reference to a Dataset resource containing the request to provision storage
```

Comment:
like the comment above, we have a runtime backward compatibility issue.

Reply:
Restored the field